### ISSUE TITLE
Optimizer for unet model on Pascal Voc segmentation

### ISSUE THREAD

**Issue by saiphanish7** [warmspringwinds/pytorch-segmentation-detection#24]
Hello,
Can I know the optimizer and its specifications to use on unet model on Pascal Voc segmentation using FOCAL loss ?
Should I have to use any learning rate schedulers?
Also, is it better to take mean focal loss or sum focal loss ?
I am training the model from scratch.
